string input_string
---
string updated_string